app/assets/stylesheets/index.scss

```diff
@@ -7,7 +7,13 @@
   padding-left: 10px;
   padding-top: 20px;
   padding-bottom: 0;
-  margin-left: 10px;
+  margin-bottom: 10px;
+  margin-left: 20px;
+  margin-right: 20px;
+  background-color: $light-blue;
+  &:hover{
```

Comment:
Rule declaration should be preceded by an empty line<br>Opening curly brace `{` should be preceded by one space